src/test/java/marquez/client/MarquezClientTest.java

```diff
@@ -106,7 +105,7 @@
   private static final Instant NOMINAL_START_TIME = newTimestamp();
   private static final Instant NOMINAL_END_TIME = newTimestamp();
   private static final String RUN_ARGS = newRunArgs();
-  private static final RunState RUN_STATE = RunState.NEW;
```

Comment:
Should we delete RunState.java?

Reply:
Yes! Just checked in the change that removes `RunState`